frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -7,6 +7,5 @@
 <br />
 
 <div class="form-buttons" data-hook="buttons">
-  <%= submit_tag Spree.t(:place_order), :class => 'continue button primary' %>
-  <script>Spree.disableSaveOnClick();</script>
+  <%= render :partial => 'spree/shared/place_order', :locals => { :order => @order } %>
```

Comment:
If you only use the `place_order` partial only at a single place, then it's better to do without it, as it only introduces a level of indirection. Also the `spree/checkout/..` path would've been a better place then `spree/shared..`